metadata-ingestion/src/datahub/ingestion/source/unity/proxy.py

```diff
@@ -196,61 +213,73 @@ def _query_history(
                 method, path, body={**body, "page_token": response["next_page_token"]}
             )
 
-    def list_lineages_by_table(self, table_name: str) -> dict:
+    def list_lineages_by_table(
+        self, table_name: str, include_entity_lineage: bool
+    ) -> dict:
         """List table lineage by table name."""
         return self._workspace_client.api_client.do(
             method="GET",
-            path="/api/2.0/lineage-tracking/table-lineage/get",
-            body={"table_name": table_name},
+            path="/api/2.0/lineage-tracking/table-lineage",
```

Comment:
For some reason, the `/get` endpoint returns the key "upstream_tables" while the base endpoint returns the key "upstreams". To stay consistent with the [docs](https://docs.databricks.com/en/data-governance/unity-catalog/data-lineage.html) I swap to the base endpoint